src/components/DrillDownTableLinkedCell/tests/index.test.tsx

```diff
@@ -0,0 +1,104 @@
+import { mount } from 'enzyme';
+import { createBrowserHistory } from 'history';
+import React from 'react';
+import { Router } from 'react-router-dom';
+import { CellInfo } from 'react-table';
+import DrillDownTableLinkedCell from '..';
+
+const urlPath: string = 'http://example.com/path';
+/* tslint:disable:object-literal-sort-keys */
+
+const cell: CellInfo = {
+  index: 0,
+  viewIndex: 0,
+  pageSize: 20,
+  page: 0,
+  level: 0,
+  nestingPath: [0],
+  column: {
+    Header: '',
+    show: true,
+    minWidth: 90,
+    className: '',
+    style: {},
+    headerClassName: '',
+    headerStyle: {},
+    footerClassName: '',
+    footerStyle: {},
+    filterAll: false,
+    id: 'id',
+  },
+  value: '2019-11-20',
+  resized: [],
+  show: true,
+  width: 90,
+  tdProps: {
+    rest: {},
+  },
+  columnProps: {
+    rest: {},
+  },
+  classes: [],
+  styles: {},
+  pivoted: false,
+  expander: false,
+  isExpanded: false,
+  maxWidth: 4,
+  row: {},
+  rowValues: {},
+  aggregated: false,
+  groupedByPivot: false,
+  subRows: [],
+  original: {},
+};
+/* tslint:disable:object-literal-sort-keys */
+
+/**
+ * Investiate why cellValue = 'some value' fails but
+ * cellValue = cell.value works. Also why should either work if the component expects the value to be
+ * of type Node?
+ */
+const cellValue = cell.value;
+const history = createBrowserHistory();
+
+describe('/components/DrillDownTableLinkedCell', () => {
+  const props = {
+    cell,
+    cellValue,
+    hasChildren: true,
+    urlPath,
+  };
+  const wrapper = mount(
+    <Router history={history}>
+      <DrillDownTableLinkedCell {...props} />
+    </Router>
+  );
+
+  it('receives props correctly', () => {
+    expect(wrapper.find(DrillDownTableLinkedCell).props()).toEqual(props);
+  });
+
+  it('renders correctly if prop hasChildren is true', () => {
+    expect(wrapper.find(DrillDownTableLinkedCell).html()).toEqual(
+      '<div><a href="/http://example.com/path/2019-11-20">2019-11-20</a></div>'
```

Comment:
@kelvin-muchiri do you have any insight on why this is happening? i.e. adding a `/` to the URL.  Do we need to create an issue to ensure that this component can actually successfully render links to external URLs?

Reply:
Well, that's a nice thought. I understand in react linking to an external URL you'll have to use anchor tag and not `Link`. Maybe we should update the test and use urlPath say as `/path` to avoid confusion.

Reply:
@kelvin-muchiri not sure what you meant, but try it out and lets see